front/src/reducers/osrdconf/osrdConfCommon/__tests__/utils.ts

```diff
@@ -155,15 +155,15 @@ const testCommonConfReducers = (slice: OperationalStudiesConfSlice | StdcmConfSl
   });
 
   it('should handle updateElectricalProfileSetId with number', () => {
-    const newElectricalProfileSetId = 1;
-    defaultStore.dispatch(slice.actions.updateElectricalProfileSetId(newElectricalProfileSetId));
+    const newElectricalProfileSetID = 1;
```

Comment:
You still have some unwanted changes in this file
```suggestion
    const newElectricalProfileSetId = 1;
```